test_solar_open_accuracy.py

```
@@ -0,0 +1,611 @@
+
+def send_slack(webhook_url: str, message: str):
+    """Send a Slack notification."""
+    import urllib.request
+    import json as _json
+
+    payload = _json.dumps({"text": message}).encode("utf-8")
+    req = urllib.request.Request(
+        webhook_url,
+        data=payload,
+        headers={"Content-Type": "application/json"},
+    )
```

Ruff (0.15.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)